api/elastic_search.py

```diff
@@ -28,9 +29,16 @@ def insert(self, doc):
     self.index(self.DEFAULT_INDEX, self.DEFAULT_TYPE, body=doc)
 
   def search_for(self, word, language="English"):
+    if len(language) == 2 or len(language) == 3:
```

Comment:
Since elasticsearch currently has full language names, converting it here if the incoming language is a 2- or 3-character code.